posthog/temporal/batch_exports/squash_person_overrides.py

```diff
@@ -468,131 +260,172 @@ async def squash_events_partition(inputs: QueryInputs) -> None:
     """
```

Comment:
I don't think the first item in the list in the docstring above is applicable any longer.

Reply:
Updated this in latest commit together with other docstring.